src/Abstractions/TaskOptions.cs

```diff
@@ -14,7 +14,7 @@ public record TaskOptions
     /// Initializes a new instance of the <see cref="TaskOptions"/> class.
     /// </summary>
     /// <param name="retry">The task retry options.</param>
-    public TaskOptions(TaskRetryOptions? retry = null)
+    public TaskOptions(TaskRetryOptions? retry)

```

Comment:
You could also change this to:


```suggestion
    public TaskOptions(TaskRetryOptions? retry)
      : this(retry, null)
    {
    }
```